yearbook/students/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class Student(models.Model):
+    name = models.CharField(max_length=100)
+    student_class = models.CharField(max_length=50)
+    email = models.EmailField(unique=True)
+    photo = models.ImageField(upload_to='photos/')
+
+    def __str__(self):
+        return self.name
+
```

Comment:
Consider adding a Class Meta to list order of student based on class

Reply:
Thank you